database/migrations/2023_12_11_202944_create_floor_price_history_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Models\Collection;
+use App\Models\Token;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('floor_price_history', function (Blueprint $table) {
```

Comment:
Does it make sense for `floor_price` and `token_id` to be nullable? Couldn't we instead just not store anything if we don't have either of them?

Reply:
@oXtxNt9U yes, good catch